README.MD

```diff
@@ -7,20 +7,25 @@ docker-compose up -d
 ```
 
 # Executing a task
-When your docker setup is running you can execute a task using `run_task.py`. Replace the value in `IMAGE` with your 
+When your docker setup is running you can execute a task using `run_task.py`. Replace the value in `IMAGE` with your
 desired docker image
 
 # Docker registry
 For development purposes you might want to run your own local docker registry:
 ```
 # Run docker registry separately. Node tasks are run in a network separate from the node itself. By making the registry
 # available from the host network it will be accessable by all docker containers
-docker run -d  --name registry --network host registry:2 
+docker run -d  --name registry --network host registry:2
 ```
 
+# How to test things
+## Using the `run_task.py` command
 The `run_task.py` script runs a custom algorithm [available on github](https://github.com/CARRIER-project/vantage6-algorithms)
 Make sure this task is published to the local registry by running `push_to_local_registry.sh` from that repo
 
 ```
 python run_task.py
 ```
+## Using the `test_federated_learning.ipynb` notebook
+If you run docker-compose up docerk will automatically spin up a jupyter notebook server for you. You can use the
```

Comment:
small typo

Reply:
Never heard of `docerk`? it's the next new thing!